js/scorm/wrapper.js

```diff
@@ -540,19 +536,21 @@ class ScormWrapper {
 
   async handleError(error) {
     if (!Data.isReady) await Data.whenReady();
-    // defer error to allow other plugins which may be handling errors to execute first
-    _.defer(() => {
-      if ('value' in error.data) {
-        // because some browsers (e.g. Firefox) don't like displaying very long strings in the window.confirm dialog
-        if (error.data.value.length && error.data.value.length > 80) error.data.value = error.data.value.slice(0, 80) + '...';
-        // if the value being set is an empty string, ensure it displays in the error as ''
-        if (error.data.value === '') error.data.value = '\'\'';
-      }
-      const config = Adapt.course.get('_spoor');
-      const messages = Object.assign({}, ScormError.defaultMessages, config && config._messages);
-      const message = Handlebars.compile(messages[error.name])(error.data);
-      switch (error.name) {
-        case CLIENT_COULD_NOT_CONNECT:
+    if ('value' in error.data) {
```

Comment:
Why has the defer moved?

Reply:
I think it was because the defer was only added to prevent multiple notifications from being opened at once, e.g. if [adapt-trackingErrors](https://github.com/adaptlearning/adapt-contrib-trackingErrors) is enabled, so it doesn't need to delay the other parts of the function. Happy to revert if necessary as I can't remember another reason why I changed it and can't replicate any potential issues as a result of it remaining on line 544 instead.

Reply:
Yes, please revert unless it's related to the issue.